frontend/src/app/user.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@angular/core';
+import { Headers, Http, Response, RequestOptions } from '@angular/http';
+import { Observable } from 'rxjs/Rx';
+
+@Injectable()
+export class UserService {
+  private url = 'http://localhost:8000/api/login';
+
+  constructor (
+    private http: Http
+  ) {}
+
+  loginUser(username: string, password: string) {
+    let headers: Headers = new Headers({
```

Comment:
`const` instead of `let`

BTW we should create our own HttpService, that will add headers in generic way